src/System.Security.Cryptography.Algorithms/tests/PKCS1MaskGenerationMethodTest.cs

```diff
@@ -32,6 +32,7 @@ namespace System.Security.Cryptography.Algorithms.Tests
     public class PKCS1MaskGenerationMethodTest 
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Net46)]
```

Comment:
I think `TargetFrameworkMonikers.NetFramework` is more accurate (since it's all versions)